packages/stacks-svelte/src/components/EmptyState/EmptyState.stories.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts" module>
+    import { defineMeta } from "@storybook/addon-svelte-csf";
+    import EmptyState from "./EmptyState.svelte";
+    import Link from "../Link/Link.svelte";
+
+    const { Story } = defineMeta({
+        title: "Components/EmptyState",
+        component: EmptyState,
+        tags: ["labs"],
+        argTypes: {
+            title: {
+                control: "text",
+            },
+            class: {
+                control: "text",
+            },
+        },
+    });
+</script>
+
+<Story name="Base" args={{ title: "No items found" }}>
+    {#snippet template(args)}
+        <EmptyState title={args.title} class={args.class}>
+            {#snippet description()}
+                There are no items to display at this time.
+            {/snippet}
+            {#snippet callToAction()}
+                <button class="s-btn s-btn__filled">Add New Item</button>
```

Comment:
We can use the `Button` component here to dogfood + get a little simpler markup:

```suggestion
                <Button variant="tonal">Add New Item</Button>
```

Reply:
Fixed 👍🏾